pkg/scraper/bruneida_scraper.go

```diff
@@ -17,21 +19,28 @@ func ScrapeBruneida() []model.Job {
 	jobCollector := linkCollector.Clone()
 
 	jobs := []model.Job{}
+	links := []string{}
+	jobIds := []string{}
 
 	// Scraping the links
 	linkCollector.OnHTML("ul.list-az.ul-azs", func(e *colly.HTMLElement) {
 		e.ForEach(".az-detail>h3.az-title>a.h-elips", func(i int, child *colly.HTMLElement) {
 			link := child.Attr("href")
+
 			jobCollector.Visit(link)
+			links = append(links, link)
 
+			jobId := getBruneidaJobId(link)
+			jobIds = append(jobIds, jobId)
 		})
 	})
 
 	// Scraping the jobs
 	jobCollector.OnHTML("body", func(h *colly.HTMLElement) {
-		job_title := h.ChildText("#title-box-inner div.inline-block.pull-left h1")
+		jobTitle := h.ChildText("#title-box-inner div.inline-block.pull-left h1")
 		company := h.ChildText("#ad-contact ul li:first-child span.bb b.small")
 		salary := h.ChildText("#ad-body-inner .opt .opt-dl:nth-child(3) .dd")
+		description := util.StandardizeSpaces(h.ChildText("#full-description"))
```

Comment:
Can you add a TODO note here that this should be solved by using a HTML minifier and whitelist?

I think that would be able to solve the problem of potentially unsafe scripts and also get rid of the extra white spaces.